### PR TITLE
Update Ionic4 Instructions

### PR DESCRIPTION
**Pull request by davidpatrick**
In line with https://github.com/auth0-samples/auth0-ionic4-samples/pull/6 these instructions fix a logout issue with Ionic 4 that doesn't close the in app browser after logout.

### CODE REVIEW

File: articles/quickstart/native/ionic4/01-login.md
Comment (lbalmaceda) [inline articles/quickstart/native/ionic4/01-login.md]: I think the auth service should not expose the safari view controller directly. Instead, a new method `hideBrowser` or similar could be created and the property remain _private_. 

Also, after checking the sample code, wouldn't it be better to call `hide()` after you get an event from that browser you just opened? if that's something that you receive and can relate to this scenario..

See 

https://github.com/auth0-samples/auth0-ionic4-samples/pull/6/files#diff-fab356ab6389b09c66727fb44caa49ccL80-L87

**Review by lbalmaceda — CHANGES_REQUESTED** [review]
I'm sure this works but I think it can be improved and encapsulate fields on their respective classes/files.

**Comment by lbalmaceda** [comment]
On a separate note, why did codeowners not kick in for this PR? https://github.com/auth0/docs/blob/master/.github/CODEOWNERS#L1

File: articles/quickstart/native/ionic4/01-login.md
Comment (davidpatrick) [reply articles/quickstart/native/ionic4/01-login.md]: I agree @lbalmaceda , I had toyed with that solution, but opted to default to how the Ionic 3 sample we had was handling it.  After getting a second pair of eyes on it, definitely think we should go the route you are suggesting

**Review by lbalmaceda — APPROVED** [review]

**Pull request merged**
